rc/extra/ranger.kak

```diff
@@ -23,6 +23,18 @@ All the optional arguments are forwarded to the ranger utility} \
             kak -p $kak_session; \
             tmux select-pane -t $kak_client_env_TMUX_PANE'.format(file=fm.thisfile.path)) \
           if fm.thisfile.is_file else fm.execute_console('move right=1')"
+  elif [ -n "$kak_client_env_STY" ]; then
+    screen -X eval \
+        'split -h' \
+        'focus down' \
+        'screen ranger $@ --cmd " \
+            map <return> eval \
+                fm.execute_console('shell \
+                    echo evaluate-commands -client $kak_client edit {file} | \
```

Comment:
Prefer using `printf %s\\n` instead of `echo`, to avoid portability issues.

Wrap `$kak_client` with double quotes as well.

Reply:
@lenormf Shell quoting is driving me insane. Unfortunately I don't know how to do this correctly so that if ${kak_client} or ${kak_session} have spaces in them, they are handled.  Can you offer a fix?

Reply:
Maybe this would work:

```
echo \\\"evaluate-commands -client \\\\\"${kak_client}\\\\\" edit {file}\\\" | kak -p \\\"${kak_session}\\\"
```

Reply:
> @lenormf: Maybe this would work [...]

Does that work for you?  Unfortunately, I can't get it to work.

Reply:
@lenormf Would it make sense to write `ranger.kak` in python given that running ranger implies that there's a python interpreter available?  That would at least help avoid all the shell-quoting woes, I think.